src/components/MyAccountInput/MyAccountInputStyled.js

```diff
@@ -39,6 +40,11 @@ export const InputElementStyled = styled.input.attrs(() => ({
   font-size: 13px;
   line-height: 13px;
 
+  ${isRTL() &&
+    css`
+      text-align: right;
```

Comment:
`text-align: start` and you can get rid of this 😄 https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties/Floating_and_positioning